include/ui/hexeditwidget.h

```diff
@@ -26,6 +26,7 @@
 #include <QToolButton>
 #include <QMainWindow>
 #include <QSharedPointer>
+#include <QWidgetAction>
```

Comment:
Please add newline after this section of includes (yes, it was wrong before ;) ).